tools/dockerfile/interoptest/grpc_interop_go/Dockerfile_1.7

```diff
@@ -0,0 +1,51 @@
+# Copyright 2015, Google Inc.
```

Comment:
Do you mean a grpc_interop_go/grpc_interop_go1.7 as the filename or grpc_interop_go1.7/Dockerfile as separate directory?